menu.py

```diff
@@ -331,7 +334,7 @@ def draw(self):
                     pyxel.rect(21, 11, 158, 182, pyxel.COLOR_BLACK)
                     pyxel.text(85, 15, "Factions", pyxel.COLOR_WHITE)
                     pyxel.text(25, 30, str(self.faction_colours[self.faction_wiki_idx][0].value),
-                            self.faction_colours[self.faction_wiki_idx][1])
+                        self.faction_colours[self.faction_wiki_idx][1])
```

Comment:
I think this has actually made it worse but it's fine lol

Reply:
oh no i forgor 💀

Reply:
[c8f4e20](https://github.com/ChrisNeedham24/microcosm/pull/91/commits/c8f4e200da76f53b0aea67e242812e7d0ff8f24d)